submitit/helpers.py

```diff
@@ -308,11 +314,17 @@ def clean_env() -> tp.Iterator[None]:
     with submitit.helpers.clean_env():
         executor.submit(...)
     """
+    if extra_names is None:
+        extra_names = ()
     distrib_names = ("MASTER_ADDR", "MASTER_PORT", "RANK", "WORLD_SIZE", "LOCAL_RANK", "LOCAL_WORLD_SIZE")
```

Comment:
you could just set `forbidden_names = ("MASTER_ADDR",...) + tuple(extra_names)` then use it, that seems simpler than the 2 `or`. Then again, it's about the same so whichever seems best to you

Reply:
There is no performance concern here, so I think keeping the two lists separate is better for clarity